api/exception_handling.py

```diff
@@ -92,12 +92,6 @@ def custom_exception_handler(exc, context):
             response = Response(data=exc.__dict__, status=status.HTTP_400_BAD_REQUEST)
         # CASE: # Other uncaught error that should not be displayed to the client
         else:
-            response = Response(
-                data=ProjectAPIException(
-                    global_error="Une erreur inatendue est survenue, veuillez réessayer plus tard.",
-                    log_level=logging.ERROR,
-                ).__dict__,
-                status=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            )
+            return None
```

Comment:
Un petit commentaire pour remplacer "# CASE: # Other uncaught error that should not be displayed to the client" ?

Reply:
Tout à fait :+1:

Reply:
https://github.com/betagouv/complements-alimentaires/pull/559/commits/6a1a9e4593505c960e046488995f97b5c5bd11b8